extensions/schema-compare/src/dialogs/schemaCompareDialog.ts

```diff
@@ -288,10 +288,12 @@ export class SchemaCompareDialog {
 		if (isTarget) {
 			this.targetFileButton = view.modelBuilder.button().withProperties({
 				label: '•••',
+				ariaLabel: localize('schemaCompare.selectFileAriaLabel', "Select file")
```

Comment:
These should probably be more descriptive, like "Select target file" and "Select source file" so someone can differentiate between them.